OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2876,6 +2876,12 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				if (engineVersion < 20160111)
```

Comment:
It is already the bottom most one. This way I can separate it from https://github.com/OpenRA/OpenRA/pull/10364.